cloud/azure/modules/app/logs.tf

```diff
@@ -15,6 +15,7 @@ resource "azurerm_monitor_diagnostic_setting" "app_service_log_analytics" {
     category = "AppServiceAppLogs"
 
     retention_policy {
+      days    = 0
```

Comment:
question (blocking): why default these all to zero?

Reply:
ahh this is a bug in terraform azure these aren't enabled so it shouldn't matter but we have to default them to false with days=0 https://github.com/hashicorp/terraform-provider-azurerm/issues/7235 otherwise it shows that there are terraform changes when you try to apply when there were no terraform changes